coralproject/talk
# Switching e2e to port 3011.

## What does this PR do?

Switched e2e to run on port 3011 to avoid conflicts with de environments.

## How do I test this PR?

1. Start a dev server on a port other than 3011 (3000 is default.)
2. `npm run e2e`
3. All tests should pass.
